src/models/tshirt/Tshirt.cpp

```diff
@@ -121,4 +121,4 @@ namespace tshirt {
         return 0;
     }
 
-}
\ No newline at end of file
+}
```

Comment:
Why is this a "change"?

Reply:
I am not sure how that got in there actually. Is it not part of the namespace from the top? I wasn't able to finish compiling this due to the make_unique from 4.8.5.

Reply:
Oh I think this just means no newline at end of file. I think it must be a gedit thing from opening the file.